dll/apisets/CMakeLists.txt

```diff
@@ -30,7 +30,9 @@ function (add_apiset apiset_name baseaddress)
         target_compile_options(${apiset_name} PRIVATE /wd4026 /wd4273)
     endif()
 
-    add_importlibs(${apiset_name} ${ARGN} ntdll)
+    # link against our "raw" import libs, otherwise add_importlibs brings msvcrtex in
+    list(TRANSFORM ARGN PREPEND lib OUTPUT_VARIABLE liblist)
+    target_link_libraries(${apiset_name} ${liblist} libntdll)
```

Comment:
There are also some other macro's called in `add_importlibs`, namely:
`add_dependency_node(${_module})`, `add_dependencies(${_module} lib${LIB})`, `add_dependency_edge(${_module} ${LIB})`
Those are not needed?

Reply:
`msvcrtex` is only brought in on `msvcrt`, so would it make sense to add an optional argument on `add_importlibs` to specify that msvcrtex should be skipped?

Reply:
right, we should probably take this, too

Reply:
`add_dependency_node` and `add_dependency_edge` are needed only for building an xml file with dependencies (we're bypassing that in so many places - does it make sense at all?)
As of `add_dependencies`, it seems that it is not needed when used with `target_link_libraries`: https://stackoverflow.com/questions/27282938/target-link-libraries-and-add-dependencies

Giving all that, do we need `add_importlibs` at all?

Reply:
Fixed :-)